base/abstractarray.jl

```diff
@@ -1256,6 +1256,9 @@ end
 
```

Comment:
How about specifying `A::AbstractArray` on the line above while we're at it? The description below isn't guaranteed to match non-array objects.

Reply:
that page is specific to array interface, i think thats implied